torch/csrc/jit/runtime/slice_indices_adjust.cpp

```diff
@@ -0,0 +1,56 @@
+#include <torch/csrc/jit/runtime/slice_indices_adjust.h>
+#include <c10/util/Exception.h>
+#include <cstdint>
+
+namespace torch {
+namespace jit {
+
+int64_t PySlice_AdjustIndices(
```

Comment:
Though this is copied from Python, we can change its name to something more appropriate for our use case right?